llama_index/storage/docstore/dynamodb_docstore.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import Optional
+from llama_index.storage.docstore.keyval_docstore import KVDocumentStore
+from llama_index.storage.kvstore.dynamodb_kvstore import DynamoDBKVStore
+
+
+class DynamoDBDocumentStore(KVDocumentStore):
```

Comment:
This is great! Would you be able to add a tutorial notebook to showcase an example of using these classes? (We usually link these into our readthedocs documentation too)

Reply:
I have never used notebook at all, but I will try to write it based on existing ones.

Reply:
Tutorial notebook added.
- `examples/docstore/DynamoDBDocstoreDemo.ipynb`